charts/kube-state-metrics/chart/values.yaml

```
@@ -0,0 +1,6 @@
+kube-state-metrics:
+  ## set to true to add the release label so scraping of the servicemonitor with kube-prometheus-stack works out of the box
+  # releaseLabel: false
+  prometheus:
+    monitor:
+      enabled: true
```

With the current setup, a serviceMonitor needs the `instance: primary` label to be picked up by the grafana-agent.

We could look into if it is plausible to remove the label requirement?

Source: https://github.com/distributed-technologies/helm-charts/blob/c045310d795d01d6388eb5136c77be7edfc5e826/charts/grafana-agent/chart/templates/metrics-instance.yaml#L14-L16

```suggestion
      enabled: true
      additionalLabels:
        instance: primary
```